drivers/auxdisplay/auxdisplay_serlcd.c

```diff
@@ -413,7 +407,8 @@ static const struct auxdisplay_driver_api auxdisplay_serlcd_auxdisplay_api = {
 
 #define AUXDISPLAY_SERLCD_INST(inst)                                                               \
 	static const struct auxdisplay_serlcd_config auxdisplay_serlcd_config_##inst = {           \
-		.capabilities = {                                                                  \
+		.capabilities =                                                                    \
+			{                                                                          \
```

Comment:
this, restore back to:

```c
		.capabilities = {                                                                  \
```

Reply:
But that's how .clang-format in zephyr formats it - see all the auxdisplay files - they all look the same

Reply:
clang-format is known to mess up these definitions, the coding guideline is same as linux: structure instantiations go with the bracket inline with the declaration and field indented one tab. Sorry some reviewers don't check those so there's counter examples in the code, then if your change changes them back in compliance ok, but in this case you are changing it to non compliant AND it's unrelated to your change.

So if you could change it so it's either compliant or as it was before, that would be ok

Reply:
> You keep insisting on a change - which all the other files in the same directory are looking different and your clang-format insists on doing the same as the other files.

Yes, I know, I'm sorry, clang-format is known to do changes against the coding style in these cases, I always flag them if I see them.

Reply:
I've manully reverted the line.

Butif not clang-format which one should I use (I'm not keen on manually formatting files...)?

Reply:
Okay I see. Unfortunately we never got clang-format to fully comply withe coding guidelines for code nested into macros, plus most files are not clang-formatted in the first place, so format on save is not really going to work for upstream work, I'm sorry.

If a file is clang-formatted already you can add `/* clang-format off */` and `/* clang-format on */` on the part where it messes up, and then at least you could rerun clang-format as you wish after that, however, most upstream files are not clang-formatted in the first place and it's normally not accepted to not reformat code unrelated to the change which is why I asked to revert this).

If a file is not clang formatted and you want to clang-format it, it may be ok to do it on a separate commit (as long as you block out cases like the one we discussed here), the maintainer may or may not be happy with that (I would not for the areas I maintain), but I recommend against that because even after you reformatted it, since we don't have any enforcing in CI and since most other contributors indent manually, eventually it's just going to be off again.